### ISSUE TITLE
Usersnap Feedback - Is it possible to have an org quick submit button somewhere in this sectio[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#815]
**Sender**: scottowenfletcher@gmail.com
**Comment**: Is it possible to have an org quick submit button somewhere in this section? maybe if you type in the name and nothing's in the database, the drop down selection could be 'Can't find your organization? Enter it now!'
[Open #512 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/512)

<a href='https://usersnappublic.s3.amazonaws.com/2019-08-07/19-23/8deb71af-e3eb-40e1-9504-ac6b80e6b5bc.png'>![Usersnap Feedback - Is it possible to have an org quick submit button somewhere in this sectio(...)](https://usersnappublic.s3.amazonaws.com/2019-08-07/19-23/ab2a560d-1250-47c1-b5e5-c65ebd186bd7.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-08-07/19-23/8deb71af-e3eb-40e1-9504-ac6b80e6b5bc.png'>Download original image</a>

**Browser**: Chrome 76 (Windows 10)
**Referer**: [https://participedia.net/case/5994/edit?full=1](https://participedia.net/case/5994/edit?full=1)
**Screen size**: 1280 x 720 **Browser size**: 1920 x 774

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by ascott** [comment]
@scottofletcher @jesicarson we could add a link to this instructional text to the new org quick submit page

**Comment by scottofletcher** [comment]
works for me!

**Comment by ascott** [comment]
rethinking this change since we've just had all the french translations verified. @scottofletcher @plscully do you think this is a priority change?

**Comment by jesicarson** [comment]
I’m not sure I understand the change? What do you mean by a “link to this instructional text on the new org quick sub page?”

**Comment by plscully** [comment]
I can see how this would be very useful for people like @scottofletcher who does lots of editing, but my sense is that we should hold off on this change until we have a better understanding of how more typical users respond to the forms. In the meantime, let's add it to P2 or beyond so that we don't lose track of the idea.

**Comment by ascott** [comment]
sorry that wasn't very clear. i meant we could add a link to the instructional text that would link to the edit page for a new organization.

**Comment by ascott** [comment]
i'll remove the p1 label for now and keep the copy-changes label so we can review later.

**Comment by scottofletcher** [comment]
I agree that it isn't a P1 priority, but I think it would be used by more people than just me. in fact, I don't think I'd be the primary user - it would be all the folks who are entering a case, get to that field, find that their organization hasn't been created yet, and then don't go through the rigamarole of publishing, quick submitting the org, publishing, going back to the case, getting to the full length edit screen, and scrolling down and entering in the org's name. so it would improve user experience and insentivize the creation of more organization entries (which, because they are linked to case entries in the data, 'deepen' our datasets - something Edana et al. requested)

**Comment by jesicarson** [comment]
> add a link to the instructional text that would link to the edit page for a new organization.

@ascott Are there any implications to publishing two entries at the same time? Would publishing an organization while an un-published case is open in another window or tab cause any problems? My main concern is people losing data, but there's also the issue of people potentially being confused about what happened to the case they were working on if they're taken to a new, blank form. 

If we aren't worried about the above, then adding that link seems like a super simple fix, and we can probably ask the french folks to very quickly re-verify that string as needed.

**Comment by scottofletcher** [comment]
I usually have about 4 ppedia.net tabs open at the same time and I often publish things simultaneously without any problems. I do notice that there's a bit of a lag between when I publish an entry and when it becomes available as a selection in the case entry field. for example, I'll be editing a case, get to the org field, find out that the org doesn't yet have an entry, open a new ppedia.net tab, write/publish an org, go back to the case entry, hit publish -> refresh page -> edit -> full version, scroll down to org field, start typing org name. at that point it seems to depend on how long it's been since I published the org. usually 5 mins is enough
